scripts/webpack/webpack-resources.js

```diff
@@ -5,22 +5,22 @@
  * @typedef {WebpackConfig & { devServer?: object }} WebpackServeConfig
  * @typedef {import("webpack").Entry} WebpackEntry
  * @typedef {import("webpack").Module} WebpackModule
- * @typedef {import("webpack").Output} WebpackOutput
  */
-/** */
+
```

Comment:
Please bring this line back--it's not super important but it prevents intellisense from associating all the typedef comments above with the webpack import.
```suggestion
/** */
```